### PR TITLE
python-packages: added geeknote with dependencies

### PR DESCRIPTION
**Pull request by hbunke**
geeknote, thrift, markdown2, evernote

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -14994,4 +14994,83 @@ let
     };
   };
 
+  
+  markdown2 = buildPythonPackage rec {
+    name = "markdown2-${version}";
+    version = "2.3.0";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/markdown2/markdown2-2.3.0.zip";
```
Comment (edwtjo) [inline pkgs/top-level/python-packages.nix]: you can reuse ${name} here

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -14994,4 +14994,83 @@ let
...
+      md5 = "8e19e848c3fcfbd7424ffedcadecbd46";
```
Comment (edwtjo) [inline pkgs/top-level/python-packages.nix]: please use sha256, see nix-hash

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -14994,4 +14994,83 @@ let
...
+    };
+    propagatedBuildInputs = with self; [];
+    meta = {
```
Comment (edwtjo) [inline pkgs/top-level/python-packages.nix]: missing license and maintainer

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -14994,4 +14994,83 @@ let
...
+      description = "A fast and complete Python implementation of Markdown";
+      homepage =  https://github.com/trentm/python-markdown2;
+    };
+  };
+  
+  
+  evernote = buildPythonPackage rec {
```
Comment (edwtjo) [inline pkgs/top-level/python-packages.nix]: the same as above

File: pkgs/top-level/python-packages.nix [hunk 5]
```diff
@@ -14994,4 +14994,83 @@ let
...
+
+     propagatedBuildInputs = with self; [ oauth2 ];
+
+     meta = {
+      description = "Evernote SDK for Python";
+      homepage = http://dev.evernote.com;
+     };
+  };
+    
+  thrift = buildPythonPackage rec {
```
Comment (edwtjo) [inline pkgs/top-level/python-packages.nix]: same as above

File: pkgs/top-level/python-packages.nix [hunk 6]
```diff
@@ -14994,4 +14994,83 @@ let
...
+
+    doCheck = false;
+
+    meta = {
+      description = "Python bindings for the Apache Thrift RPC system";
+      homepage = http://thrift.apache.org/;
+    };
+  };
+  
+  geeknote = buildPythonPackage rec {
```
Comment (edwtjo) [inline pkgs/top-level/python-packages.nix]: same as above except the name reuse

File: pkgs/top-level/python-packages.nix [hunk 7]
```diff
@@ -14994,4 +14994,83 @@ let
...
+    version = "7ea2255bb6";
+    name = "geeknote-${version}";
+    disabled = ! isPy27; 
+
+    src = pkgs.fetchgit {
```
Comment (edwtjo) [inline pkgs/top-level/python-packages.nix]: please use fetchFromGitHub

**Comment by hbunke** [comment]
done and squashed. thanks for guiding a newbie.

File: pkgs/top-level/python-packages.nix [hunk 8]
```diff
@@ -14994,4 +14994,94 @@ let
...
+  markdown2 = buildPythonPackage rec {
+    name = "markdown2-${version}";
+    version = "2.3.0";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/markdown2/${name}.zip";
+      sha256 = "073zyx3caqa9zlzxa82k9k2nhhn8c5imqpgp5nwqnh0fgaj9pqn8";
+    };
+    propagatedBuildInputs = with self; [];
+    meta = {
```
Comment (edwtjo) [inline pkgs/top-level/python-packages.nix]: your metas' rhs scope doesn't contain licenses or maintainers. you need to bring them in either by being explicit or adding `with stdenv.lib;` to the attrset declaration.

File: pkgs/top-level/python-packages.nix [hunk 9]
```diff
@@ -14994,4 +14994,94 @@ let
...
+  thrift = buildPythonPackage rec {
+    name = "thrift-${version}";
+    version = "0.9.2";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/t/thrift/${name}.tar.gz";
+      sha256 = "1yla6wg18x2a0l0lrvkp1v464hqhff98ck8pnv8d5j9kn3j6bxh8";
+    };
+
+    doCheck = false;
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: Just a quick note: please add a short comment why tests are disabled

**Comment by hbunke** [comment]
I love how you guys are educating me here ;-) Thanks a lot!

**Comment by hbunke** [comment]
OK. I've also seen 'git-YYYY-MM-DD'. Which one is better?

**Comment by edwtjo** [comment]
IMHO skip the git part

**Comment by jagajaga** [comment]
@hbunke @edwtjo don't skip the `git-version` part, change it as @codyopel said.

**Comment by edwtjo** [comment]
Thank you!

**Pull request merged**